apps/rule-manager/client/src/ts/utils/camelCaseToTitleCase.ts

```diff
@@ -0,0 +1,5 @@
+export const camelCaseToTitleCase = (str: string) => {
+    // Add space between strings
+    const result = str.replace(/([A-Z])/g,' $1');
```

Comment:
Ingenious! But could we replace with lodash's `startCase`?

Reply:
Smashing, let's see how they did it:
```
const startCase = (string) => (
  words(`${string}`.replace(/['\u2019]/g, '')).reduce((result, word, index) => (
    result + (index ? ' ' : '') + upperFirst(word)
  ), '')
)
```

Reply:
Crikey 😀